src/base/memory_monitor_cl.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_cl.cpp - definitions of client functions
+ *                         for memory monitoring module
+ */
+
+#include <stdio.h>
+#include <cstring>
+#include <string>
+#include <cassert>
+
+#include "memory_monitor_cl.hpp"
+
+/*
+ * mmon_convert_module_name_to_index -
+ *
+ * return: module index
+ *
+ *   module_name(in):
+ */
+int
+mmon_convert_module_name_to_index (const char *module_name)
+{
+  for (int i = 0; i < MMON_MODULE_LAST; i++)
```

Comment:
```suggestion
  for (int i = -1; i < MMON_MODULE_LAST; i++)
```
what if module_name is `ALL` ? is it possible?

Reply:
Because we change the spec to get integer value in module option, so we don't think about it. This function is just used for printing.